### PR TITLE
Starting precheck tool.

### PR DESCRIPTION
**Pull request by crunk1**
This precheck tool, for now, only checks OS Versions against a list of GCE official supported OSes.

### CODE REVIEW

File: import_precheck/check_osversion.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	if goos == "windows" {
+		return c.getWindowsInfo()
+	}
+
+	out, err := exec.Command("cat", "/etc/redhat-release").Output()
+	if err == nil {
+		return string(out), err
+	}
+
+	out, err = exec.Command("cat", "/etc/issue").Output()
```
Comment (zmarano) [inline import_precheck/check_osversion.go]: Look at /etc/os-release instead

File: import_precheck/check_osversion.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	"github.com/crunk1/golang-sak"
+)
+
+var OSVERSIONS = []string{
+	"Windows Server 2008 R2",
+	"Windows Server 2012 R2",
+	"Windows Server 2016 R2",
+	"Debian GNU/Linux 8",
+	"Debian GNU/Linux 9",
```
Comment (zmarano) [inline import_precheck/check_osversion.go]: Add Ubuntu 14.04, 16.04

File: import_precheck/check_osversion.go (hunk 1 above)
Comment (adjackura) [reply import_precheck/check_osversion.go]: Actually this is more complicated than that. Remind me on Monday to share the code I have for the inventory agent.

Comment (adjackura) [reply import_precheck/check_osversion.go]: Also ioutil.readfile

File: import_precheck/check_osversion.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		return r, nil
+	}
+
+	contains := func(i interface{}) interface{} {
+		if strings.Contains(info.Version, i.(string)) {
+			return i
+		}
+		return nil
+	}
+	if version := sak.Any(versions, contains); version == nil {
```
Comment (adjackura) [inline import_precheck/check_osversion.go]: This function is super simple, just reimplement instead of pulling in a thirdparty source

**Comment by crunk1** [comment]
Got this ready. @adjackura @zmarano mind taking a look?

**Pull request merged**